docs/Signing.md

```diff
@@ -6,20 +6,23 @@ and information about the strongname or certificate that should be used.
 
 ## Format
 
-The signing request manifest supports three element types. A minimal example looks like this. See [Elements](#Elements) below for details
+The signing request manifest supports multiple element types. Each element represents a specific kind of action to take on a file. A minimal example looks like this. See [Elements](#Elements) below for details
 
 ```xml
 <SigningRequest>
   <File Path="MyAssembly.dll" Certificate="MyCert" StrongName="MyStrongName" />
```

Comment:
Where is the path rooted? `$(artifactsDir)/build`?

Reply:
Rooted next to the signrequest.xml file.